ExtendedControls/Themes/ThemeStandard.cs

```diff
@@ -949,6 +975,14 @@ public bool IsFontAvailableInTheme(string themename, out string fontwanted)
             return false;
         }
 
+        public bool IsFontAvailable(string fontwanted)
```

Comment:
Not a huge fan of this approach. Think that a full font replacement mapping, or the simplicity of falling back to MS Sans Serif, would be better. Now, can no longer select the fancy themes unless the fonts are installed, which means I'm not aware that other themes exist that want fonts, which means that I'm not looking for the fonts to download to try out the cool "new" theme.